### PR TITLE
rbenv: add page

### PR DESCRIPTION
**Pull request by thezeroalpha**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/rbenv.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+- Show the currently selected Ruby version:
+
+`rbenv version`
+
+- Uninstall a version of Ruby:
+
+`rbenv uninstall {{version}}`
+
+- List Ruby versions that contain an executable program:
```
Comment (agnivade) [inline pages/common/rbenv.md]: What does "contain an executable program" mean ? I guess you mean to say what is the version of a ruby binary ? Won't that be shown easily by `ruby -v` ? In that case, I don't think this example is much beneficial to be in a tldr page.

Comment (pxgamer) [reply pages/common/rbenv.md]: It displays all ruby versions that contain the specified executable. Say you have a Gem on multiple versions of Ruby, then it will display these. For example, `rbenv whence travis` would display which versions of Ruby one has Travis installed for. Afaik.

Comment (thezeroalpha) [reply pages/common/rbenv.md]: Yes, @pxgamer is right, that's exactly what it does. Should I reword it to make that clearer?

Comment (thezeroalpha) [reply pages/common/rbenv.md]: Well, @pxgamer's description also includes the word "contain", but I agree that it's better. I'll change it.

Comment (agnivade) [reply pages/common/rbenv.md]: I meant this - 
> For example, rbenv whence travis would display which versions of Ruby one has Travis installed for. Afaik.

**Comment by agnivade** [comment]
Going to merge this one, and send a new PR with something that I feel might be clearer.
